analysis_options.yaml

```diff
@@ -33,6 +33,7 @@ analyzer:
     # Please see https://github.com/flutter/flutter/pull/24528 for details.
     sdk_version_async_exported_from_core: ignore
   exclude:
+    - 'example/**'
```

Comment:
**suggestion:** Let's not edit this file, and keep the `example` within the included files.

Reply:
I updated the deprecated property in `example` and kept it included in the `analyzer` as is. We can re-visit this area in a separate PR later that addresses linting.